package.json

```diff
@@ -20,6 +20,7 @@
   },
   "homepage": "https://github.com/Lv-237-NodeJS/da-mi#readme",
   "devDependencies": {
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
```

Comment:
They recommend in dev - https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy